.prettier.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  overrides: [
+    {
+      files: "**/*.sol",
+      options: {
+        printWidth: 120,
+        tabWidth: 2,
+      },
+    },
+    {
+      files: ["**/*.js", "**/*.ts", "**/*.vue"],
+      options: {
+        bracketSpacing: true,
+        trailingComma: "es5",
+        tabWidth: 2,
+        printWidth: 120,
```

Comment:
We recently decided that 100 is what we use for solidity projects, let's match that here too (umbra will prob stay at 120 which I think is where this was from)
```suggestion
        printWidth: 100,
```